src/cache/cache.js

```diff
@@ -149,7 +149,10 @@ module.exports = function(router) {
         const formRevision = rev.revision || rev.formRevision;
         debug.loadSubForms(`Loading form ${util.idToBson(rev.form)} revision ${formRevision}`);
         const loadRevision = formRevision.length === 24 ? router.formio.resources.formrevision.model.findOne(
-            {_id: util.idToBson(rev.revision)}
+          {$or: [
+            {_id: util.idToBson(rev.revision)},
```

Comment:
Why is this $or needed here?  When is the revision ID ever saved as the _id?  It makes me nervous that it can be either the _id or the revisionId...  If it should always be "revisionId", then isn't the change here to just change _id to revisionId?